engines/ejs-engine/src/index.js

```diff
@@ -31,10 +31,17 @@ const engine = {
  * @return {string} EJS component string
  */
 window.component = (path, props) => {
+  	// Handle that the full path is not provided (just component-name, e.g, component("button"))
+	const modifiedPathParts = path.split('/');
+	const lastPart = modifiedPathParts[modifiedPathParts.length - 1];
+  const newPath = `${path.toLowerCase().replace('.ejs', '')}/${lastPart}.ejs`;
```

Comment:
Looks like lastPart will make it through with case preserved, you might want to lowercase that too

Reply:
Right, good spotting

Reply:
Fixed in https://github.com/CloudCannon/bookshop/pull/18/commits/a030c2a32dd3cd0e65792ebd6cf7fd57bc8939ee :)